agent/src/util/device_plugin_service.rs

```diff
@@ -115,7 +116,9 @@ impl DevicePlugin for DevicePluginService {
         Ok(Response::new(resp))
     }
 
-    type ListAndWatchStream = mpsc::Receiver<Result<ListAndWatchResponse, Status>>;
+    type ListAndWatchStream =
+        Pin<Box<dyn Stream<Item = Result<ListAndWatchResponse, Status>> + Send + Sync + 'static>>;
+    // type ListAndWatchStream = mpsc::Receiver<Result<ListAndWatchResponse, Status>>;
```

Comment:
remove commented line.

( ick ... mpsc::Receiver looks so much more understandable :( )

Reply:
I agree :( Fixed in 67595e71132f

Reply:
it looks like someone got ReceiverStream to compile without the Box/Pin stuff ... might simplify the code?

https://github.com/massbitprotocol/test-ci-for-massbitprotocol/blob/80698ad273eb56d8e168b7c34671292e6e9e7615/chain-reader/chain-reader/src/grpc_stream.rs

Reply:
good point. fixed in 5dc53e223be4e6a29a6f4310625872d5cbc8215d